crates/task-impls/src/consensus.rs

```diff
@@ -343,6 +343,11 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                 "Updating view from {} to {} in consensus task",
                 *self.cur_view, *new_view
             );
+
+            if *self.cur_view / 100 != *new_view / 100 {
+                info!("Progress: entered view {:>6}", *new_view);
```

Comment:
Can we make this a `error` log instead? I know it _should definitely_ be info, and we have this on the backlog, but: basically we only ever run with warn or error because we have a _lot_ of debug and info logs.

Reply:
Sure!